### PR TITLE
Updated Distro pom.xml With New module versions for Ref App 2.7 and D…

### PR DESCRIPTION
**Pull request by tendomatGitHub**
@rkorytkowski  and  @dkayiwa  can you please take a look at this  and see if the changes are correct ?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -144,21 +141,6 @@
 			</dependency>
 
 			<!-- API dependencies -->
-                            <dependency>
```
Comment (ssmusoke) [inline pom.xml]: @tendomatGitHub Why have you removed these dependencies?

**Comment by ssmusoke** [comment]
@tendomatGitHub Can you squash your commits

**Comment by tendomatGitHub** [comment]
@ssmusoke  okk working on that. looks like the latest commit has no conflict

**Comment by tendomatGitHub** [comment]
@ssmusoke  the branch has conflicts,i don't know why this is like so now.Because i pulled latest changes before editing.and the build was successful with "mvn clean install " and "mvn clean package"

**Comment by dkayiwa** [comment]
I remember you did this in another pull request. So am closing this.

**Comment by tendomatGitHub** [comment]
That's ok.I had actually deleted the whole repo and started over again,because for some unknown reason the commits could not be squashed.

**Pull request closed without merging**